00_Base/.eslintrc.js

```diff
@@ -1,7 +1,10 @@
 /* eslint-env node */
 module.exports = {
-  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
-  parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
   root: true,
-};
\ No newline at end of file
+  rules: {
```

Comment:
this change can be ignored as it will be addressed in the prettier/eslint PR but TS was complaining about an explicit any and I figured I would just adjust it here